### ISSUE TITLE
Impossible to add a plugin as described in the documentation

### ISSUE THREAD

**Issue by LcsGa** [ckeditor/ckeditor5-angular#397]
On an angular 16 app (a pristine project as it was intended to be a POC for me to learn how ckeditor works), I tried to follow the the tutorial step by step.
Everything went smoothly, until I reached the [Commands](https://ckeditor.com/docs/ckeditor5/40.0.0/tutorials/crash-course/commands.html) section.

The issue I'm facing is that I get this error :
![image](https://github.com/ckeditor/ckeditor5-angular/assets/58547290/bac47f17-6f9b-4f26-b2a6-b05a5c736691)

Then I followed the [documentation](https://ckeditor.com/docs/ckeditor5/latest/support/error-codes.html#error-ckeditor-duplicated-modules) to fix it. Thus I modified my imports, from :
![image](https://github.com/ckeditor/ckeditor5-angular/assets/58547290/e2521484-fbc9-4283-910d-2fd28d3745ca)
to :
![image](https://github.com/ckeditor/ckeditor5-angular/assets/58547290/514c50c3-599f-49ea-8e76-016abb005e7b)

Once I did so, the error was indeed gone but a new one appeared :
![image](https://github.com/ckeditor/ckeditor5-angular/assets/58547290/600c7a0f-3701-44e9-a6fd-e45ab9c97789)

Now I don't know what other fixes I should try...

In my app I tried multipled things, from using the ckeditor angular module/component with my custom plugin, but also with a ckeditor plugin and the issue remained the same.

Then I tried to make my own ckeditor component but once again, the issue was the same !

moreover, when I try to build the project, I receive many errors on icons:
- with esbuild
![image](https://github.com/ckeditor/ckeditor5-angular/assets/58547290/1bbb5b80-b8ac-40e0-9bbc-c2fe59806c8e)
- with webpack
![image](https://github.com/ckeditor/ckeditor5-angular/assets/58547290/77106536-2b71-4e0e-8ee5-f07c23bf76af)

Btw, my dependencies are not clean as I have peer deps but since it was just for a little poc, I ignored them and force installed the deps. I should maybe investigate there but would love to hear your thoughts about it before losing more time :) !

Here is the [repo](https://github.com/LcsGa/ckeditor) of the poc

**Comment by Witoso** [comment]
Hey, it's not possible to extend [the predefinded builds](https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/predefined-builds.html#when-not-to-use-predefined-builds) with plugins, you need to [create a custom build](https://ckeditor.com/docs/ckeditor5/latest/installation/getting-started/quick-start-other.html).

**Comment by LcsGa** [comment]
Yep, that's what I tried to do after posting that message but I ended up having the same issues on a custom build...

=> I forked the "classic build" and added either already existing plugins and custom ones but the issue was with the existing ones this time, if I remember correctly (like adding CodeBlock for exemple)

**Comment by Witoso** [comment]
Hmm, the tutorial uses `@ckeditor/ckeditor5-editor-classic` which is a different package.

Have you checked the guide for [Angular](https://ckeditor.com/docs/ckeditor5/latest/installation/integrations/angular.html#integrating-a-build-from-the-online-builder)?

**Comment by LcsGa** [comment]
Yes, really carefully! it didn't really help me

**Comment by Witoso** [comment]
Could pack up the project for us to take a look when we have time? cc @mremiszewski

**Comment by LcsGa** [comment]
I could try yep, but not right away! I'll do it as soon as possible
